tool/embedded-binaries.ts

```diff
@@ -61,20 +61,26 @@ export async function getDartSassEmbedded(outPath: string): Promise<void> {
     // the Github API instead of manually piecing together the asset URL. The
     // API exposes only the latest *non*-pre-release versions.
     // https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#get-the-latest-release
-    let response = await fetch(
-      'https://api.github.com/repos/sass/dart-sass-embedded/releases'
-    );
-    if (!response.ok) throw Error(response.statusText);
-    const latestRelease = JSON.parse(await response.text())[0];
+    //
+    // Mock the latest release version for now. Travis can't call the Github API
+    // because Github seems to be throttling all calls from the osx instance.
```

Comment:
Is there a way to set up a repo token to get around this? It seems like all anonymous requests by Travis's osx instances to the Github API are using the same IP address, or otherwise saturating the API.

(Keep in mind that this script needs to be run by both Travis and any contributors developing locally.)

Reply:
I've added a `GITHUB_AUTH` environment variable that's visible on Travis and has SassBot's authorization token. If you add that to Basic Auth with the username "sassbot" that should work. Locally, users will be using different IPs so they're unlikely to run up against rate-limiting.

[This is the logic we use for GitHub requests in Dart Sass](https://github.com/sass/dart-sass/blob/4f1f5c9de80499ab2f4bface7dbda2d3ef50a41e/tool/travis/sass-spec-ref.sh#L25-L32), for reference.